prospector/README.md

```diff
@@ -82,6 +82,15 @@ NOTE: this is about the "legacy" CLI. A new CLI is being developed.
 
 `python main.py <vulnerability_id> -r <repository_url> -v`
 
+## Testing
+
+First, you need to install pytest-cov:
+
+`pip install pytest-cov`
```

Comment:
@amilankovich-slab good idea to mention the tests, however the best thing is to modify the part where the readme says
`pipenv install` and replace it with `pipenv install --dev` (because pytest-cov is in the Pipfile already, as a dev dependency).

FYI: the --dev flag installs "normal" deps + dev deps